### DISCUSSION TITLE
Changed bundle structure after update gatsby

### DISCUSSION THREAD

**Discussion by ostaoleho** [gatsbyjs/gatsby#37682] [Help]
I have site on gatsby after update gatsby from 3v to 4v bundle structure changed. I followed the instruction - [https://www.gatsbyjs.com/docs/reference/release-notes/migrating-from-v3-to-v4/](https://www.gatsbyjs.com/docs/reference/release-notes/migrating-from-v3-to-v4/). 

Before was this
<img width="1795" alt="Screenshot 2023-02-16 at 20 46 43 (1)" src="https://user-images.githubusercontent.com/78026498/220373380-290b942e-6083-4169-ad8c-49ba1146cea7.png">

And after update I got this
<img width="1796" alt="Screenshot 2023-02-21 at 16 54 33" src="https://user-images.githubusercontent.com/78026498/220378987-f7100618-9974-4bc1-a037-fdd08f47233f.png">

Then I added webpack configs  

` config.optimization.splitChunks = {
    chunks: 'all',
    name: false,
    cacheGroups: {
      vendors: {
        test: /[\\/]node_modules[\\/]/,
        reuseExistingChunk: true,
      },
      commons: {
        minChunks: 2,
        reuseExistingChunk: true,
      },
    },
  };`

and the result was 
<img width="1793" alt="Screenshot 2023-02-16 at 20 46 01 (1)" src="https://user-images.githubusercontent.com/78026498/220374447-2fcb6036-d77c-4bf7-a5b9-acc5546fcc39.png">

I was able to achieve almost the same result using React.lazy but had problems with css loads
**Can I get the same chunks with gatsby 4 as I got with 3rd version? (without lazy load)**

**Comment by grahamd711** [comment]
I'm experiencing a similar issue when upgrading from Gatsby 3 to Gatsby 4. Is there anything in the build process that has changed? Our page speed scores were negatively impacted due to increase in bundle size.
